frontend/src/app/pages/exercises/exercise/shared/exercise-map/shared/transfer-point-popup/transfer-point-popup.component.ts

```diff
@@ -44,6 +45,14 @@ export class TransferPointPopupComponent implements PopupComponent, OnInit {
 
     public hospitals$ = this.store.select(selectHospitals);
 
+    public getOrderByValueTransferPoints: (
```

Comment:
Either 
```suggestion
    public getOrderByValueTransferPoint: (
```

```suggestion
    public getTransferPointOrderByValue: (
```

- the names are still not pretty, but I'm unable to come up with better ones.